hera_pspec/grouping.py

```diff
@@ -386,6 +405,14 @@ def average_spectra(uvp_in, blpair_groups=None, time_avg=False,
                     wgts = uvp.get_wgts(spw, blp, p)
                     ints = uvp.get_integrations(spw, blp, p)[:, None]
                     w = (ints * np.sqrt(nsmp))
+
+                    # Get error bar weights and set invalid ones to zero.
```

Comment:
because you don't use `errs` further down, just set as `errws` to being with so there is no confusion.

What abour errs that are != -99 but are < 0? Seems like the third line should actually read
```
errs['errs < 0'] = 0.0
```